platform/authorized_api/tracker_settings/public_v2.yaml

```diff
@@ -361,7 +361,7 @@ components:
                 enable_fingerprinting_across_websites:
                   $ref: '#/components/schemas/EnableFingerprintingAcrossWebsites'
                 set_ip_tracking:
-                  $ref: '#/components/schemas/SetIpTracking'
+                  $ref: '#/components/schemas/SetIpTracking'  
```

Comment:
additional spaces?

Reply:
yeah it is automatically generated from https://github.com/PiwikPRO/PPMS-ApiDocs and should be fixed also there, here fixed